### DISCUSSION TITLE
Support for LNURL Verify

### DISCUSSION THREAD

**Discussion by AnthonyRonning** [btcpayserver/btcpayserver#6057] [Ideas / Feature Requests]
[LUD-21](https://github.com/lnurl/luds/blob/luds/21.md) allows for an LN Address server to also provide feedback when the invoice was paid. This is one of the connections that Zaprite uses for their invoices and it works rather well. Allows for all kinds of lightning backend connections, from custodians like Alby to mobile wallets like Mutiny.

We've gotten a request to allow someone's Mutiny wallet be the lightning destination for a BTCPayServer invoice. Without needing to do a "custom" integration with Mutiny, this would be possible if the invoice flow was based on LNURL Address + LNURL Verify.

I do not know the complete stack of BPS so I'm not sure how feasible this would be, just throwing it out there.

**Comment by john-zaprite** [comment]
Great timing! We've just received a request for this too from a BTCPay user.

**Comment by NicolasDorier** [comment]
Shipped in 2.3.7 https://github.com/btcpayserver/btcpayserver/issues/7248
